src/quoridor/controller/gamecontroller.go

```diff
@@ -19,7 +19,7 @@ func CreateGame(conf *game.Configuration) (*game.Game, error) {
 	lineCenter := (boardSize - 1) / 2
 	pawn := game.Pawn{game.Position{0, lineCenter}}	
 	id:= shortuuid.New()
-	game := game.Game{id, pawn, []game.Fence{}, board}
+	game := game.Game{id, false, pawn, []game.Fence{}, board}
```

Comment:
There's a lot of args but you can't use named arg using Golang... =(

https://stackoverflow.com/questions/23447217/go-named-arguments